lib/HackTest.rb

```diff
@@ -22,9 +23,11 @@ def my_handler(_event_type, _headers, data)
     ) # TODO: handle pagination
     non_inclusive_words_found = search_files(files)
 
-    if data_hash.key?(:pull_request) && (PR_ACTIONS.include? data_hash[:action])
-      handle_pull_request(data_hash, non_inclusive_words_found)
-    end
+    return unless data_hash.key?(:pull_request) && PR_ACTIONS.include?(data_hash[:action])
+
+    return if non_inclusive_words_found.empty?
```

Comment:
So here I was thinking of maybe posting a comment of “hey your PR has passed the check” if this is empty?

Reply:
Hmm, I mean we could log something here to keep track, but I don't think we want to be commenting on every PR? I feel like people would get annoyed at that lol.